rust/kcl-lib/src/parsing/parser.rs

```diff
@@ -2821,6 +2834,7 @@ fn fn_call_kw(i: &mut TokenSlice) -> PResult<Node<CallExpressionKw>> {
         alt((
             terminated(non_code_node.map(ArgPlace::NonCode), whitespace),
             terminated(labeled_argument, labeled_arg_separator).map(ArgPlace::LabeledArg),
+            terminated(expression, (opt(whitespace), equals)).map(ArgPlace::LabelWithoutRhs),
```

Comment:
This can backtrack a lot. Are you sure you want to do this?

Reply:
Maybe this can be merge with the line below.

Reply:
This is the performance difference (I ran on my branch then on main, so treat the "performance gains" as performance losses). 3-4% slowdown on the really long programs is OK IMO.

<img width="1143" alt="Screenshot 2025-03-11 at 12 48 38 PM" src="https://github.com/user-attachments/assets/a578cb3c-1cf1-4637-9065-d304b12481ef" />

Reply:
OK I merged it with the line below.